L1Trigger/L1TMuonEndCap/plugins/emtf-geometry-gen.cc

```diff
@@ -0,0 +1,1240 @@
+#include <cmath>
+#include <fstream>
+#include <string>
+#include <sstream>
+#include <iomanip>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <L1Trigger/L1TMuonEndCap/plugins/emtf-geometry-gen.h>
+
+#define UPPER_THETA 45
+#define LOWER_THETA 8.5
+
+
+
+//the boolean isTMB07 is set to true, but I still don't know what it's for.
+
+using namespace std;
+
+slhc_geometry::slhc_geometry(edm::ParameterSet const& conf)
```

Comment:
could we give this class a CMS complaint name?
(and what does "slhc" mean in this context?)